.dotnet/src/Custom/Chat/ChatCompletion.cs

```diff
@@ -40,6 +41,11 @@ public partial class ChatCompletion
         ? Choices[0].Logprobs.Content
         : _contentTokenLogProbabilities ??= new ChangeTrackingList<ChatTokenLogProbabilityInfo>();
 
+    // CUSTOM: Flattened refusal logprobs property.
+    public IReadOnlyList<ChatTokenLogProbabilityInfo> RefusalTokenLogProbabilities => (Choices[0]?.Logprobs != null)
```

Comment:
When I named the other property "ContentTokenLogProbabilities", I remember thinking "What other token log probabilities could there even be?", but I decided to keep the name just because. And here we are today.